src/Monolog/Logger.php

```diff
@@ -285,9 +285,11 @@ public function addRecord($level, $message, array $context = array())
         foreach ($this->processors as $processor) {
             $record = call_user_func($processor, $record);
         }
-        while (isset($this->handlers[$handlerKey]) &&
```

Comment:
Yes, I had read that. What is "bubbling"? Maybe it is just because I am no English native speaker, but I think that is far from clear =)